src/main/java/core/basesyntax/service/impl/FruitParserImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitShopTransactions;
+import core.basesyntax.service.FruitParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitParserImpl implements FruitParser {
+    private static final int TITLE_INDEX = 0;
+    private static final String DATA_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    private FruitShopTransactions.Operation getOperationByLetter(String inputOperation) {
+        for (FruitShopTransactions.Operation operation : FruitShopTransactions.Operation.values()) {
+            if (operation.getOperation().equals(inputOperation)) {
+                return operation;
+            }
+        }
```

Comment:
Also, you can try to use map here

Reply:
Speaking of the map, what do you mean? Do you mean use threads instead of loops?

Reply:
No, i'm talking about HashMap. It is a good replacement for the `if` or `switch` blocks. Also, you can think about how to fill up the map when the app started and then reuse it in this method

Reply:
But it is optional, your solution is Ok)